src/elmExecutor/buildPopulatedResourceBundle.js

```diff
@@ -1,5 +1,35 @@
-function doSearch(smart, type, callback) {
+function doSearch(smart, type, fhirVersion, request, callback) {
   const q = {};
+  
+  // setup the query for Practitioner and Coverage
+  switch (type) {
+    case "Practitioner":
+      q._id = request.performer.reference;
```

Comment:
won't this crash the app if the request doesn't have a performer?  We should null-proof this.

`request.performer && request.performer.reference`

Or we could get fancy and use null propagation.

```suggestion
      q._id = request?.performer?.reference;
```